src/components/DeliverableView/DocumentsUploader.tsx

```diff
@@ -3,18 +3,25 @@ import { useCallback, useMemo, useState } from 'react';
 import { Box } from '@mui/material';
 import { FileChooser } from '@terraware/web-components';
 
+import StatusChangeConfirmationDialog from 'src/components/DeliverableView/StatusChangeConfirmationDialog';
 import { useLocalization } from 'src/providers';
 import strings from 'src/strings';
 
 import FileUploadDialog from './FileUploadDialog';
 import { ViewProps } from './types';
 
 type DocumentUploaderProps = ViewProps & {
+  deliverableStatusesToIgnore: string[];
```

Comment:
Can you make this `DeliverableStatusType[]` so we have type safety?

Reply:
I actually meant to change that but forgot, you got it 👍